src/main/components/styled-components/Clock/Clock.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8" ?>
 
 <component name="Clock" extends="Group" >
+  <interface>
+    <field id="fontUrl"   type="String"   alwaysNotify="true"   onChange="onDisplayLabelFontChanged" />
+    <field id="fontSize"  type="Integer"  alwaysNotify="true"   onChange="onDisplayLabelFontSizeChanged" />
```

Comment:
out of curiosity, why use `alwaysNotify="true"` here?

Reply:
Also, for fontSize, do you think it would be useful to put a default `value` here?

Reply:
> out of curiosity, why use `alwaysNotify="true"` here?

It will notify all the changes to the `CallBack` funtion

Reply:
> Also, for fontSize, do you think it would be useful to put a default `value` here?

agree!